docs/guides/admin/docs/configuration/ui.md

```diff
@@ -77,3 +77,43 @@ Available Language Configuration
 
 The admin UI is translated into a number of languages by default.  If you wish to restrict the languages available to
 your users, add the relevant locale code to `etc/org.opencastproject.adminui.endpoint.LanguageServiceEndpoint.cfg`.
+
+
+Configuring the events publisher metadata
+-----------------------------------------
+
+the metadata can be used in two ways, and its meaning varies slightly:
+
+* The publisher is the creator of the event: when an event is created, this field is filled automatically with the
+logged in user. It cannot be modified, neither in the creation nor in the modification.
+* The publisher is responsible for uploading the content but may not be the creator of the event in the UI:
+in this case, when the event is created, the publisher is selected from a list provider plus the logged in user
+(selected by default) and it is also modifiable later in the properties edition (but in edition, the logged in user
```

Comment:
I would leave out "in the properties edition" and change the () part to "but then the logged-in user is not selectable"